extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/AccessResource.scala

```diff
@@ -27,6 +27,7 @@ import org.apache.kyuubi.plugin.spark.authz.OperationType.OperationType
 
 class AccessResource private (val objectType: ObjectType) extends RangerAccessResourceImpl {
```

Comment:
How about `case class AccessResource (objectType: ObjectType, catalog: Option[String])

Reply:
OK, addressed. The `catalog` is not suitable for setting in `elements` of RangerAccessResourceImpl, or it causes unexpected unauthorization results.